### PR TITLE
Onboarding project into team CI Server.

### PR DESCRIPTION
**Pull request by stephenashank**

### CODE REVIEW

File: jenkins/saveAndCompress.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+mkdir result
+
+# The files will only exist if maven reaches the corresponding phase of the build
+cpe() { [[ ! -e $1 ]] || cp -r -t result $1; }
```
Comment (craigdbarber) [inline jenkins/saveAndCompress.sh]: This function is concise, but is hard to read.
```suggestion
function cpe() { 
  if [[ -e $1 ]]; then
    cp -r $1 result
  fi 
}
```

**Comment by craigdbarber** [comment]
Also, it looks like there's some tests which still need to be debugged in the pipeline.

File: jenkins/saveAndCompress.sh (hunk 1 above)
Comment (stephenashank) [reply jenkins/saveAndCompress.sh]: Done

File: Jenkinsfile [hunk 2]
```diff
@@ -1 +1,52 @@
...
+            yamlFile 'jenkins/maven-pod.yaml'
+        }
+    }
+
+    environment {
+        GOOGLE_PROJECT_ID = "${GCE_IT_PROJECT_ID}"
+        GOOGLE_REGION = "${GCE_IT_REGION}"
+        GOOGLE_ZONE = "${GCE_IT_ZONE}"
+        GOOGLE_SA_NAME = "${GCE_IT_SA}"
+        FAILED_ARTIFACTS_BUCKET = "${GCE_IT_BUCKET}"
```
Comment (craigdbarber) [inline Jenkinsfile]: Suggest renaming this, since the artifacts will be archived every build now.

**Comment by craigdbarber** [comment]
Looks like the saveAndCompress.sh script needs to be given +x

**Comment by craigdbarber** [comment]
Looks good. Thanks for taking care of this!

**Pull request merged**